_opensearch/query-dsl/bool.md

```diff
@@ -9,18 +9,20 @@ nav_order: 45
 
 You can perform a Boolean query with the `bool` query type. A Boolean query compounds query clauses so you can combine multiple search queries with Boolean logic. To narrow or broaden your search results, use the `bool` query clause rules.
 
-As a compound query type, `bool` allows you to construct an advanced query by chaining together several simple ones.
+As a compound query type, `bool` allows you to construct an advanced query by combining several simple queries.
 
 Use the following rules to define how to combine multiple sub-query clauses within a `bool` query:
 
 Clause rule | Behavior
 :--- | :---
-`must` | The results must match the queries in this clause. If you have multiple queries, every single one must match. Acts as an `and` operator.
-`must_not` | This is the anti-must clause. All matches are excluded from the results. Acts as a `not` operator.
-`should` | The results should, but don't have to, match the queries. Each matching `should` clause increases the relevancy score. As an option, you can require one or more queries to match the value of the `minimum_number_should_match` parameter (default is 1).
-`filter` | Filters reduce your dataset before applying the queries. A query within a filter clause is a yes-no option, where if a document matches the query it's included in the results. Otherwise, it's not. Filter queries do not affect the relevancy score that the results are sorted by. The results of a filter query are generally cached so they tend to run faster. Use the filter query to filter the results based on exact matches, ranges, dates, numbers, and so on.
+`must` | Logical `and` operator. The results must match the queries in this clause. If you have multiple queries, all of them must match.
+`must_not` | Logical `not` operator. All matches are excluded from the results.
+`should` | Logical `or` operator. The results must match at least one of the queries, but optionally, it can match both queries. Each matching `should` clause increases the relevancy score. As an option, you can require one or more queries to match the value of the `minimum_number_should_match` parameter (default is 1).
```

Comment:
```suggestion
`should` | Logical `or` operator. The results must match at least the `minimum_number_should_match` of the queries, but optionally, it can match more queries. Each matching `should` clause increases the relevancy score. As an option, you can set the minimum number of queries that must match using the `minimum_number_should_match` parameter (default is 1).
```

Reply:
thanks for the suggestion Fanit. let's keep two separate field definitions clear.

`should` it requires one or both to match. (they can use `should` by itself as an OR operator)
`minimum_number_should_match` - optional. they can also choose to require that at least 1 must match.

Reply:
| thanks for the suggestion Fanit. let's keep two separate field definitions clear.
| should it requires one or both to match. (they can use should by itself as an OR operator)
| minimum_number_should_match - optional. they can also choose to require that at least 1 must match.

`should` requires **one or more** to match. There are two issues here: 
1. There may be not two queries, but more than two, so the word "both" does not work in this context.
2. The results must match at least one query **if** `minimum_number_should_match` is not set, so it is 1 (the default).
If the `minimum_number_should_match` is set to, let's say 2, then the results must match at least **two** queries.
So, the `minimum_number_should_match` is closely related to the `should` in that it sets the minimum number of queries in `should` the document must match to be returned in the result.

Reply:
Let's go through to clarify:

1. OR operators mean one or more. Yes. there might be more than two queries. 
Changing "both" to "more than one."

2. adding the additional explanation you provided. Thanks!